corehq/apps/users/models.py

```diff
@@ -1922,8 +1922,7 @@ def __repr__(self):
 
     @skippable_quickcache(['self._id'], lambda _: settings.UNIT_TESTING)
     def get_usercase_id(self):
-        from corehq.apps.hqcase.utils import get_case_id_by_domain_hq_user_id
-        return get_case_id_by_domain_hq_user_id(self.domain, self._id, USERCASE_TYPE)
+        return CaseAccessors(self.domain).get_case_by_domain_hq_user_id(self._id, USERCASE_TYPE)
```

Comment:
might have merged too quickly - it looks like the calling code expects an ID and this might return a case?